GVFS/GVFS.Common/GitHubUpgrader.cs

```diff
@@ -49,10 +53,14 @@ public class GitHubUpgrader : IProductUpgrader
         public GitHubUpgrader(
             string currentVersion,
             ITracer tracer,
-            GitHubUpgraderConfig upgraderConfig)
+            GitHubUpgraderConfig upgraderConfig,
+            bool dryRun = false,
+            bool noVerify = false)

```

Comment:
My only thought in looking through this is if we have more options we will want to refactor this so that we don't end up with a bunch of `bool` parameters but instead have an `UpgraderOptions` class that hold all of them including what the `GitHubUpgraderConfig` has but since we only have 2 right now I don't think we need that.